### PR TITLE
setup mocha and chai

### LINKED ISSUE #9: Setup Test Environment: mocha and Chai
** Description **
Update the existing project to use mocha as a test runner and chai as an assertion library

Acceptance Criteria
- Tests should run and fail as expected.
- Tests should reveal coverage report

### PR DESCRIPTION
**Pull request by MekkyMayata**
## Description
- installed mocha, nyc and chai dependencies.
- edit package.json to update new test script
- deleted old test configs

Fixes #9 

## How Has This Been Tested?
- on push, unit tests ran based on new setup
- Tests runs and fails as expected.
- Tests revealed coverage report.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,9722 @@
...
+  "version": "0.0.1",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "@babel/code-frame": {
+      "version": "7.8.3",
+      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.8.3.tgz",
+      "integrity": "sha512-a9gxpmdXtZEInkCSHUJDLHZVBgb1QS0jhss4cPP93EW7s+uC5bikET2twEF3KV+7rDblJcmNvTR7VJejqd2C2g==",
+      "dev": true,
+      "requires": {
```
Comment (sikiru-moshood) [inline package-lock.json:12]: Hey, kindly add package-lock.json to .gitignore

**Review by sikiru-moshood — CHANGES_REQUESTED** [review]
@MekkyMayata Kindly check the comments. Thanks!

File: package-lock.json (hunk 1 above)
Comment (MekkyMayata) [reply package-lock.json:12]: Okay, thanks.

Comment (sikiru-moshood) [reply package-lock.json:12]: Git is still seeing the commits. Could you remove it from your commit. That's why we have up to +9,772 lines of code.

Comment (sikiru-moshood) [reply package-lock.json:12]: This might help [link](https://devconnected.com/how-to-remove-files-from-git-commit/)

**Pull request closed without merging**